contents/data-display/charts/weekly-temperature-chart/metadata.json

```
@@ -0,0 +1,15 @@
+{
+  "en": {
+    "title": "Weekly Temperature Chart",
+    "description": "Displays a line graph of temperatures for one week."
+  },
+  "ja": {
+    "title": "週間気温チャート",
+    "description": "1週間の気温の線グラフを表示します。"
+  },
+  "options": {
+    "container": {
+      "p": "md"
+    }
+  }
+}
```

We can remove this `options`.

```suggestion
  },
}
```